docs/tutorials/fairplay.md

```diff
@@ -8,6 +8,13 @@ Modern EME yet, you can use legacy Apple Media Keys with:
 shaka.polyfill.PatchedMediaKeysApple.install();
 ```
 
+If you need to use both legacy and Modern EME, for example if you have to support
+multiple DRM providers, it is possible to enable uninstalling the polyfill:
+```js
+shaka.polyfill.PatchedMediaKeysApple.install(/* enableUninstall */ true);
```

Comment:
This should be: `/* enableUninstall= */ true`. This keeps it consistent with the rest of the code.

Reply:
addressed